packages/ag-charts-community/src/scene/group.ts

```diff
@@ -2,9 +2,9 @@ import { clamp } from 'ag-charts-core';
 
 import { BBox } from './bbox';
 import { HdpiOffscreenCanvas } from './canvas/hdpiOffscreenCanvas';
-import type { LayersManager } from './layersManager';
 import type { ChildNodeCounts, RenderContext } from './node';
 import { Node, SceneChangeDetection } from './node';
+import type { Scene } from './scene';
```

Comment:
Hmm... this introduces a bit of a cyclic dependency, I'd prefer if we could keep things more directed.